i2cdev-bmp280/src/lib.rs

```diff
@@ -167,6 +167,14 @@ pub fn get_linux_bmp280_i2c_device() -> Result<LinuxI2CDevice, LinuxI2CError> {
     }
 }
 
+#[cfg(any(target_os = "linux", target_os = "android"))]
+pub fn get_linux_bmp280_i2c_device_with_addr(addr: u16) -> Result<LinuxI2CDevice, LinuxI2CError> {
```

Comment:
This is a bit redundant. You could just call `LinuxI2CDevice::new("/dev/i2c-1", addr)` manually. Additionally, this doesn't have anything to do with the bmp280 anymore.